setup.py

```diff
@@ -26,23 +26,25 @@
 REQUIREMENTS = [
     "fs>=2.0.0",
```

Comment:
Yes, it should have been at least `fs >= 2.0.7` for the opener to work, so I'll change it to `fs ~= 2.0.7`.